src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,58 @@
 package mate.jdbc;
 
+import java.util.List;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static final Long EXISTING_INSTANCE = 1L;
+    private static final Long NOT_EXISTING_INSTANCE = 100L;
+    private static Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        ManufacturerService manufacturerService
+                = (ManufacturerService) injector.getInstance(ManufacturerService.class);
+
+        Manufacturer manufacturer = new Manufacturer(null, "Ford", "USA");
```

Comment:
Give more specific names
```suggestion
        Manufacturer ford = new Manufacturer(null, "Ford", "USA");
```